File: visitor/visitor.js
class Empleado {
    constructor(nombre, salario) {
        this.nombre = nombre;
        this.salario = salario;
    }

    aceptar(visitor) {
        visitor.visitar(this);
    }
}

class CalculadoraImpuestos {
    visitar(empleado) {
        const impuesto = empleado.salario * 0.15;
        console.log(`Impuesto: ${impuesto}`);
    }
}

File: visitor/visitor.py
class Empleado:
    def __init__(self, nombre, salario):
        self.nombre = nombre
        self.salario = salario

    def aceptar(self, visitor):
        visitor.visitar(self)


class CalculadoraImpuestos:
    def visitar(self, empleado):
        impuesto = empleado.salario * 0.15
        print(f"Impuesto: {impuesto}")
